### DISCUSSION TITLE
Sprint 7 Demo Outline

### DISCUSSION THREAD

**Discussion by Coffee2Bits** [DevOps-MBSE/AaC#182] [Ideas]
Since Sprint 6 was light and Sprint 7 was very research heavy, I think we should demo completed stories from both sprints and let this demo be a light one.

My thoughts on a current outline:
* AaC Package in PyPI (Maybe show downloading it and running it in a python environment?)
* Consistent plugin results
  * Improves UX
  * Makes testing easier

**Comment by lizzcondrey** [comment]
Closing and adding the resolved label since this sprint is past.
